src/components/Header/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import AppLogo from "../../assets/images/AppLogo.svg";
+import BackButton from "../../assets/images/BackButton.svg";
+import "./Header.scss";
+
+const Header = () => {
+  return (
+    <div className="header">
+      <img src={BackButton} alt="" />
+      <div className="header--container">
+        <div className="header--logo-title">
```

Comment:
both of these classnames are "elements" rather than "modifiers":

`header__container`
`header__logo-title`